butler-bot/bot.js

```diff
@@ -63,6 +68,42 @@ client.on('message', (message) => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+// Kick the user who didn't tell about himself in two weeks
```

Comment:
I would rephrase it like this
```suggestion
// Kick users who haven't introduced themselves within two weeks
```